src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  movies: Movie[];
+}
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <ul className="movies">
+    {movies.map((film: Movie) => (
```

Comment:
you can avoid typing here 
```suggestion
    {movies.map((film) => (
```

Reply:
okey, thanks